build/bk-release.sh

```diff
@@ -2,10 +2,13 @@
 
 set -euo pipefail
 
+# the user is set to avoid permission issue during the creation of the 'dist' folder
```

Comment:
maybe I'd add a comment here that the user (his `id` from `id -u`) running this command has to be added into docker group on host/runner.